src/dc_toolkit/utils.py

```diff
@@ -86,11 +87,29 @@ def open_dataset(dataset_file: str, field_to_compress: str | None = None, field_
     return ds
 
 
+def is_lat_lon(da):
+
+    lat_pattern = r'lat'
+    lon_pattern = r'lon'
+
+    dims = da.dims
+
+    if len(dims) == 2 and re.search(lat_pattern, dims[0]) and re.search(lon_pattern, dims[1]):
```

Comment:
Do you take into account the case where len(dims) == 3 and the time dim is of size 1? Because I came across this type of dataset.